lib/src/render/element_type_renderer.dart

```diff
@@ -255,6 +290,38 @@ class AliasedElementTypeRendererMd
   }
 }
 
+class AliasedFunctionTypeElementTypeRendererMd
+    extends ElementTypeRenderer<AliasedFunctionTypeElementType> {
+  const AliasedFunctionTypeElementTypeRendererMd();
+
+  @override
+  String renderLinkedName(AliasedFunctionTypeElementType elementType) {
+    var buf = StringBuffer();
+    buf.write(elementType.aliasElement.linkedName);
```

Comment:
Same here, maybe - don't re-request, provide required data.

Reply:
This could be a good idea for the renderers -- I don't like that they require passing in the higher level objects myself.  Will look into that later on.